src/cc65/main.c

```diff
@@ -951,6 +962,9 @@ int main (int argc, char* argv[])
 
                 case 'O':
                     IS_Set (&Optimize, 1);
+                    IS_Set (&CodeSizeFactor, 100);
+                    IS_Set (&EnableRegVars, 0);
+                    IS_Set (&InlineStdFuncs, 0);
```

Comment:
@groessler:

The doc explizitly says "It is possible to concatenate the modifiers for -O. For example, to enable register variables and inlining of known functions, you may use -Ors." And this is exactly what one intuitively expects. I'm having a hard time to believe that you really consider(ed) to make
`-Os -Or` mean something different than `-Osr`. This would be the opposite from all compilers I used in my life so far...